DOCSP-30519: count usage ex (#70)
mongodb/docs-rust: source/usage-examples/count.txt

.. _rust-count-usage:

===============
Count Documents
===============

You can count the number of documents in a collection by calling one of
the following methods on a ``Collection`` instance:

- ``count_documents()``: counts the number of documents that match a
  query filter
- ``estimated_document_count()``: estimates the total number of
  documents in a collection by using collection metadata

Each method returns the count as a ``u64`` instance.

.. note::
   

>    If you do not pass a filter to the ``count_documents()`` method,

Reviewer: totally does not matter but [contractions](https://www.mongodb.com/docs/meta/style-guide/style/contractions/) will make people think you are cool and laid back

```suggestion
   If you don't pass a filter to the ``count_documents()`` method,
```
